src/site/content/en/handbook/style/index.md

```diff
@@ -0,0 +1,158 @@
+
+  This sequence works well for codelabs.
+
+Focus each paragraph (typically 2–3 sentences) on one main idea.
+
+Use a list for a set of instructions or related ideas.
+
+Include examples and figures. (See the next guideline.)
+
+## Use figures when text won't do
+Aside from breaking up long sections of prose, media can be better than text for conveying some concepts and instructions. A key question to ask as you write is: "Would this idea be easier to understand if I provided a figure?"
+
+web.dev supports the following figures, which you can include in your post or codelab by following the examples in the "[web.dev components](/handbook/web-dev-components)" post.
+* Tables (Use tables sparingly—they don't resize well on mobile.)
+* Comparisons
+* Images and video (See the "[Use images and video effectively](/handbook/use-media)" post.)
+* Code blocks
+* Sample apps (See the "[Write useful code samples](/handbook/write-code-samples)" post.)
+
+## Help readers get things done
+Focus on readers and the concrete task they're trying to complete. While an introductory paragraph can provide important [context](#give-useful-context), get to the instructional content as soon as possible. The "[Use Imagemin to compress images](/use-imagemin-to-compress-images)" post is a great example.
+
+When providing instruction, favor the imperative mood.
+
+{% Compare 'worse', 'Don’t' %}
+> You can do _x_ by clicking **Run Audit**.
+
+The declarative mood ("you can…") makes it less clear that the reader needs to take an action.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> Click **Run Audit** to do _x_.
+
+The imperative "click" makes it obvious that an action is needed.
+{% endCompare %}
+
+When talking about technologies or tools, focus on what they let the reader do, not on their features or the work that went into creating them. (Avoiding first-person pronouns helps.)
+
+{% Compare 'worse', 'Don’t' %}
+> We've added _[a cool thing]_ in DevTools.
+
+This sentence emphasizes the work the developers of DevTools did rather than what it lets the reader do.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> Use _[a cool thing]_ in DevTools to _[do neat stuff]_.
+
+This sentence better focuses on what how readers can meet their goals.
+{% endCompare %}
+
+Address the reader directly rather than imagining a hypothetical third party.
+
+{% Compare 'worse', 'Don’t' %}
+> A seller can choose a variety of shopfront themes.
+
+If the reader isn't a seller, this detail may not seem relevant.
+{% endCompare %}
+
+{% Compare 'better', 'Do' %}
+> You can choose a variety of shopfront themes.
+
+Using second person helps convey that we think this information is relevant to readers. (This strategy assumes that the post has been appropriately targeted to an [audience](/handbook/audience).)
+{% endCompare %}
+
+Provide a way to measure success. How can developers test that they've followed your instruction correctly?
+
+## Keep it simple
```

Comment:
So you did deal with scope. Maybe put this immediately before or after the rule about being concise. They are two sides of the same coin.